client/src/App.js

```diff
@@ -16,9 +17,15 @@ function App() {
           <Route path="/addTour">
             <AddTour />
           </Route>
+          <Route path="/addConcurrentTour">
```

Comment:
The naming of your routes are uncommon.
Please take a look at popular websites like GitHub.

Instead, a common convention could be this:

`/tours`
`/tours/new`
`/tours/today`

If you want to distinguish between concurrent and "normal" tours, it could make sense to add a query parameter. This depends on the differences between these tours.

Search for `url design` for more information.